var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -61,19 +61,30 @@ def autoreconf(self, spec, prefix):
         aclocal('-I', 'lib/autoconf')
         autoconf()
 
+    @property
+    def optflags(self):
+        flags = '-O2'
+
+        # Optimizations for the Intel compiler, suggested by CP2K
+        if '%intel' in self.spec:
+            flags += ' -xAVX -axCORE-AVX2 -ipo'
```

Comment:
I would suggest "-xSSE2 -axAVX -axCORE-AVX2 -ipo" in case the target machine is old and doesn't support AVX.